Docker/entrypoint.sh

```diff
@@ -23,7 +23,8 @@ fi
 
 if [ -n "$OIDC_ENABLED" ] && [ "$OIDC_ENABLED" -ne 0 ]; then
 	# Debian
-	(which a2enmod >/dev/null && a2enmod -q auth_openidc) ||
+	(which a2enmod >/dev/null && a2enmod -q auth_openidc)
+ 	(echo "OIDCSessionMaxDuration ${OIDC_SESS_MAX_DURATION:-2592000}" >> /etc/apache2/mods-available/auth_openidc.conf && echo "OIDCSessionInactivityTimeout ${OIDCSessionInactivityTimeout:-86400}" >> /etc/apache2/mods-available/auth_openidc.conf) ||
```

Comment:
I believe this will still append the configuration lines multiple times, each time the container is restarted